crates/hc_run_local_services/Cargo.toml

```diff
@@ -25,7 +25,7 @@ futures = "0.3"
 holochain_trace = { version = "^0.4.0-dev.3", path = "../holochain_trace" }
 if-addrs = "0.12"
 kitsune_p2p_bootstrap = { version = "^0.3.0-dev.7", path = "../kitsune_p2p/bootstrap" }
-sbd-server = "=0.0.3-alpha"
+sbd-server = "=0.0.4-alpha"
```

Comment:
this is tested by the `run-local-services` integration test right?

Reply:
@steveej - there are currently no integration tests for run-local-services directly, but this same version bump is covered by integration/unit tests via sweetest. See https://github.com/holochain/holochain/pull/3984#pullrequestreview-2108607092